database/test_database.go

```diff
@@ -173,14 +173,19 @@ func TestSimpleKeyValueClosed(t *testing.T, db Database) {
 	require.NoError(db.Close())
 
 	_, err = db.Has(key)
-	require.Equal(ErrClosed, err)
+	require.ErrorIs(err, ErrClosed)
 
 	_, err = db.Get(key)
-	require.Equal(ErrClosed, err)
+	require.ErrorIs(err, ErrClosed)
+
+	err = db.Put(key, value)
+	require.ErrorIs(err, ErrClosed)
```

Comment:
isn't there another PR where we inline this stuff? If so, what's the policy to inline/not inline?

Reply:
We don't inline functions for `ErrorIs`. We do for `NoError`